Bottom: a0d518f6267bdf08ee99afc306ffb7a4fd257fd2
Top:    41275ebf36f106cedef6963331ebce848338db3a
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-17 01:06:44 +0100

Refresh of fix-crash-in-smashbufferelemen

---

diff --git a/src/buffer.c b/src/buffer.c
index c048a59..b396918 100644
--- a/src/buffer.c
+++ b/src/buffer.c
@@ -926,7 +926,7 @@ SmashBufferElement (BufferTypePtr Buffer)
   }
   END_LOOP;
   FreeElementMemory (element);
-  FreeElement (element);
+  g_slice_free (ElementType, element);
   return (true);
 }
